v2_1/ws/rest/docs/4_6_conneg.md

```diff
@@ -1,6 +1,10 @@
-## Selection of the Appropriate Representation
+## HTTP Content-Negotiation
 
-Selection of the appropriate formats for the response message is made using the mechanisms defined for [HTTP Content Negotiation](http://www.w3.org/Protocols/rfc2616/rfc2616-sec12.html). Using the HTTP Content Negotiation mechanism, the client specifies the desired format and version of the resource using the [Accept HTTP header](http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html).
+[HTTP Content Negotiation](http://www.w3.org/Protocols/rfc2616/rfc2616-sec12.html) is a mechanism offered by HTTP that allows clients to indicate the preferred representation, language, encoding, etc. for a resource.
+
+### Selection of the Appropriate Representation
+
+Using the HTTP Content Negotiation mechanism, the client specifies the desired format and version of the resource using the [Accept header](http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html).
 
 Along with official mime types (e.g.: text/html, application/xml, etc), the standard also defines a syntax allowing a service to define its own types. The SDMX Restful API makes use of this functionality and the syntax is as follows:
```

Comment:
Yes, we should do it, once we have an official release, as the current JSON mime types still contain an indication that it's a pre-release (e.g. application/vnd.sdmx.data+json;version=1.0.0-cts).